.github/actions/templates/avm-validateModuleDeployment/action.yml

```
@@ -55,14 +55,47 @@ inputs:
   customLocation:
     description: "Custom location overwrite, if needed"
     required: false
+  e2eignore:
```

```suggestion
  e2eIgnore:
```
given that all other parameters etc. are camel-cased...